pkg/localMetrics/localMetrics.go

```diff
@@ -0,0 +1,67 @@
+package localMetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	MetricTotalAWSAccounts = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Please only have one dummy metric here. You don't need all of these.

Reply:
done